mappings/net/minecraft/client/render/entity/EntityRendererStrayOverlay.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS none/bvz net/minecraft/client/render/entity/EntityRendererStrayOverlay
+	FIELD a OVERLAY_TEXTURE Lnone/kp;
```

Comment:
`OVERLAY_TEX` or just `TEX` if it's a simple 2D texture. Should be `SKIN` if it's applied to a 3D model.